src/Rules/ScopeRequestValidateMethods.php

```diff
@@ -0,0 +1,95 @@
+<?php declare(strict_types=1);
+
+namespace Hihaho\PhpstanRules\Rules;
+
+use Illuminate\Http\Request;
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\Php\PhpFunctionFromParserNodeReflection;
+use PHPStan\Reflection\Php\PhpParameterFromParserNodeReflection;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+use PHPStan\ShouldNotHappenException;
+
+final class ScopeRequestValidateMethods implements Rule
+{
+    public function getNodeType(): string
+    {
+        return MethodCall::class;
+    }
+
+    /** @throws ShouldNotHappenException */
+    public function processNode(Node $node, Scope $scope): array
+    {
+        if ($scope->getNamespace() === 'App\\Http\\Request') {
```

Comment:
```suggestion
        if (str_starts_with($scope->getNamespace(), 'App\\Http\\Request')) {
```

We want to allow subfolders too